### PR TITLE
First version of weekly checks config

### PR DESCRIPTION
**Pull request by kreczko**
Added `config/weekly_checks.yaml` to the project which currently analyses the efficiency of energy sums and jets.

Neither energy sum nor jets analysers currently use the new plotting functions in `cmsl1t.plotting`.

### CODE REVIEW

File: config/weekly_checks.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  pileup_file: ""
+  run_number: 276243
+
+analysis:
+  do_fit: False
+  pu_type: 0PU12,13PU19,20PU
+  pu_bins: 0,13,20,999
+  analyzers:
+    - cmsl1t.analyzers.energy_sums_efficiencies
+    - cmsl1t.analyzers.jet_efficiencies
```
Comment (benkrikler) [inline config/weekly_checks.yaml]: I've spoken in person to Alex now and said that these aren't the "contemporary" form of analyzer, and that he should base his around the `study_met` one.

File: config/weekly_checks.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  files:
+    - data/L1Ntuple_*.root
+  sample:
+    name: Data
+    title: 2016 Data
+  trigger:
+    name: SingleMu
+    title: Single Muon
+  pileup_file: ""
+  run_number: 276243
```
Comment (benkrikler) [inline config/weekly_checks.yaml]: I think it's worth putting something obviously junk-like in here so we know the person using this config should update it, and it's obvious if they forget.

Comment (kreczko) [reply config/weekly_checks.yaml]: I Agree.
The initial set of config parameters is taken from Shane's macros and needs tuning.
I want to use the weekly config 'exercise' to find these things and eliminate them bit-by-bit.

File: config/weekly_checks.yaml (hunk 1 above)
Comment (kreczko) [reply config/weekly_checks.yaml]: OK, let's change it up. I am thinking of creating a new branch `weekly_config` in here so we can do quick iterations.

Comment (benkrikler) [reply config/weekly_checks.yaml]: Good shout.  For now, based on the list of plots on issue #65, I've suggested we just have a single analyzer that runs things.  We'll have to keep an eye on how that evolves though, and potentially generalize that analyze as needs develop.

**Comment by kreczko** [comment]
BTW: What was our final verdict? Should this be merged into the new branch or does it need improvement first? I know Alex started from this branch (also includes some click fixes).

**Comment by benkrikler** [comment]
I'll merge this in and we can iterate on things later on.

**Pull request merged**